fastlane/Fastfile

```diff
@@ -0,0 +1,69 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:android)
+
+platform :android do
+  desc "Runs all the tests"
+  lane :test do
+    gradle(task: "test")
+  end
+
+  lane :tests do
+    gradle(task: "test")
+  end
+
+  lane :upload_screenshots do
+    upload_to_play_store(
+      skip_upload_apk: "true",
+      skip_upload_aab: "true",
+      version_code: "63",
```

Comment:
Do we need to specify and update version_code manually every time? Also, it's not matching the one below.